api/src/database/migrations/20210813A-add-maintenance-config.ts

```diff
@@ -0,0 +1,21 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.schema.alterTable('directus_settings', (table) => {
+		table.boolean('maintenance_active');
+		table
+			.uuid('maintenance_role')
+			.references('id')
+			.inTable('directus_roles')
+			.withKeyName('maintenance_role')
```

Comment:
Lets use our "generate index" util to make sure this aligns with other FKs we create